Added set credentials env instruction in README (#1461)
apache/polaris: README.md

- `./gradlew assemble` - To skip tests.
- `./gradlew test` - To run unit tests and integration tests.
- `./gradlew run` - To run the Polaris server locally; the server is reachable at 

>     localhost:8181. This is also suitable for running regression tests, or for connecting with Spark. 
>   - If you start Polaris via `./gradlew run`, you can set your own credentials by specifying system property `./gradlew run -Dpolaris.bootstrap.credentials=POLARIS,root,secret` where:

Reviewer: Let's make it more compacted, and remove a bullet level.
```suggestion
    localhost:8181. This is also suitable for running regression tests, or for connecting with Spark. Set your own credentials by specifying system property `./gradlew run -Dpolaris.bootstrap.credentials=POLARIS,root,secret` where:
```

Author: Thanks @flyrain !

>   - If you start Polaris as a direct java process via `./gradlew run`, you can set your own credentials using the environment variable `export POLARIS_BOOTSTRAP_CREDENTIALS=POLARIS,root,secret` where:

Reviewer 2: Unfortunately this is not technically correct. `./gradle run` sets credentials via the `-Dpolaris.bootstrap.credentials` system property, which take precedence over the env. var :shrug: 

When I said "direct java process" I mean running `java` from shell... IMO, running via Gradle is not "direct" :)

Reviewer: In that case, we should using this
```suggestion
  - If you start Polaris as a direct java process via `./gradlew run`, you can set your own credentials by specifying system property `./gradlew run -Dpolaris.bootstrap.credentials=POLARIS,root,secret` where:
```

Author: @dimas-b I removed the [ **as a direct java process** ] part.

    - `POLARIS` is the realm
    - `root` is the CLIENT_ID
    - `secret` is the CLIENT_SECRET

>   - If credentials are not set, it will use preset credentials `POLARIS,root,secret`

Reviewer 2: The `POLARIS,root,secret` default applies only to `./gradlew run`

Reviewer: This is fine, this part is only for `./gradlew run`

> - `./gradlew run` - To run the Polaris server locally; the server is reachable at 

Reviewer 2: Could you briefly mention that this run method uses preset `root:secret` credentials?

  localhost:8181. This is also suitable for running regression tests, or for connecting with Spark. 
- `./regtests/run_spark_sql.sh` - To connect from Spark SQL. Here are some example commands to run in the Spark SQL shell:

> - Set your own credentials using the environment variable `export POLARIS_BOOTSTRAP_CREDENTIALS=POLARIS,root,s3cr3t` where:

Reviewer 2: I think it might be best to disambiguate by saying `If you start Polaris as a direct `java` process with in-memory storage, set you own credentials ...`?

Reviewer: I assume this is only for `./gradlew run`. Can we move it under that bullet point so it's clear this environment variable (`POLARIS_BOOTSTRAP_CREDENTIALS`) is specifically for running Polaris locally via the Gradle task?
